src/assets.rs

```diff
@@ -308,13 +319,14 @@ impl AssetLoader for FighterLoader {
                 meta.collision_offset = meta.center_y - FOOT_PADDING;
             }
 
+            let index_len = meta.spritesheet.image.len();
             if let Some(ref mut attachment) = meta.attachment {
                 for (index, image) in attachment.image.iter().enumerate() {
                     let (texture_path, texture_handle) =
                         get_relative_asset(load_context, load_context.path(), image);
 
                     let atlas_handle = load_context.set_labeled_asset(
-                        format!("atlas_{}", index + attachment.image.len()).as_str(),
+                        format!("atlas_{}", index + index_len).as_str(),
```

Comment:
whats the reason for change here?

Reply:
This was done when the attack had a spritesheet, there's no reason for it now, gonna remove it